src/components/NavBar.js

```diff
@@ -17,14 +18,14 @@ const NavBar = () => {
   ];
   return (
     <>
-      <nav>
-        <h1>
+      <nav className={styles.NavBar}>
```

Comment:
- [ ] Please add the icon at the top-right corner of your navbar as shown in the [original design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378). 😄 🙏